blenderproc/python/constructor/RandomRoomConstructor.py

```diff
@@ -1,18 +1,23 @@
+"""
+The RandomRoomConstructor can construct a random shaped room, based on a given floor size. It also places objects
+without collision inside the created room.
+"""
+
 import warnings
 import math
 from typing import Tuple, List, Dict
+import random
 
 import bpy
 import bmesh
 import mathutils
 import numpy as np
-import random
 
 from blenderproc.python.modules.provider.getter.Material import Material
 from blenderproc.python.utility.CollisionUtility import CollisionUtility
-from blenderproc.python.types.EntityUtility import Entity, delete_multiple
+from blenderproc.python.types.EntityUtility import delete_multiple
 from blenderproc.python.types.MeshObjectUtility import MeshObject, create_primitive
-from blenderproc.python.object.FaceSlicer import FaceSlicer
+from blenderproc.python.object.FaceSlicer import _FaceSlicer
```

Comment:
Isnt that a contraction? I think FaceSlicer should be public as its used outside its own file

Reply:
I agree, I made it public.